recipe/meta.yaml

```diff
@@ -1,20 +1,17 @@
 {% set version = "0.9.24" %}
-{% set sha256 = "44602436c52c29d4f301f55f6fd8115f945469b868348e3cddaf91ab2473ea26" %}
-{% set build = 0 %}
 
 package:
   name: lmdb
   version: {{ version }}
 
 source:
-  fn: LMDB_{{ version }}.tar.gz
   url: https://github.com/LMDB/lmdb/archive/LMDB_{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 44602436c52c29d4f301f55f6fd8115f945469b868348e3cddaf91ab2473ea26
   patches:
-    - Makefile.patch
+    - patches/Makefile.patch
 
 build:
-  number: {{ build }}
+  number: 1
```

Comment:
Why is the build number bump needed? Are we changing something in the package? Looks mostly like style and metadata changes

Reply:
I tend to be careful when the infrastructure changes, which in this case, is a lot 😅 
```diff
 c_compiler_version:
-- '4'
+- '11'
```

Reply:
Ah good point!